common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1159,6 +1159,11 @@ public static String computeMaxHostBrokerProtocol() {
          */
         public static final String BROKER_REQUEST_V2_SUCCESS = "broker_request_v2_success";
 
+        /**
+         * String to send true if the request should send the PkeyAuth header to the token endpoint, false otherwise.
+         */
+        public static final String SHOULD_SEND_PKEYAUTH_HEADER_TO_THE_TOKEN_ENDPOINT = "should.send.pkeyauth.header";
```

Comment:
Should we add Silent Flow to this variable, as we are not using this in the Interactive flow